JanePhotoBrowser/PhotoBrowser/PhotoBrowserView/PhotoBrowserView.swift

```diff
@@ -39,8 +39,11 @@ public class PhotoBrowserView: UIView {
     /// The index of the current photo being shown
     public var currentPhotoIndex: Int = 0
     
+    /// The desired height of the preview collectionview
+    public var previewCollectionViewHeight: CGFloat = 0
+    
     /// Flag indicating if the preview thumbnails should show or not
-    @IBInspectable public var showPreview: Bool = false {
+    public var showPreview: Bool = false {
```

Comment:
Why’d you remove the showPreview flag?

Reply:
So when we turn it on in storyboard, it is not working as it depends on a size for the image now.   I can restructure the file to bring it back as it is setting up when it instantiates.   But since we don't want to turn it on unless we are have a specific flag on I thought I could remove it.  Let me know if we NEED it to be inspectable.

Reply:
I am sure I can bring it back, and do not instantiate the preview unless the `previewCollectionViewHeight` is greater than 0 and the `showPreview` flag is true.   It's an easy change and I think it will all still work.   Let me know what you think.

Reply:
I updated to include `@IBInspectable` for both `previewCollectionViewHeight` and `showPreview`.  So if one of the Experiments win we can just do the work in the storyboard and be done, other than the aspect ratio.